build.gradle

```diff
@@ -1,15 +1,35 @@
+buildscript {
+    repositories{
+        mavenCentral()
+        maven { url "https://jitpack.io" }
+    }
+    dependencies {
+        classpath 'com.github.qixalite:SpongeStart:1.4'
+    }
+}
+
 plugins {
-    id "com.github.hierynomus.license" version "0.12.1"
+    id 'java'
+    id 'idea'
+    id 'eclipse'
+    id 'com.github.hierynomus.license' version '0.12.1'
+    id 'net.ellune.blossom' version '1.0.1'
 }
 
+ext.spongeapi = '3.1.0-SNAPSHOT'
+ext.descriptions = 'Essence is a Sponge plugin that allows you to quickly start your server with essential commands, events, and other tidbits that you might need. '
+ext.url = 'http://essencepowered.github.io'
+
 group 'io.github.essencepowered'
-version '0.5-SNAPSHOT'
+version '0.1.0-SNAPSHOT'
```

Comment:
Why 0.1? Why not remain at 0.5? I'm inclined to remain at 0.5 because it feels closer to the finished system (if you can ever call it finished).

I guess I don't like going back in versions!